google/cloud/security/enforcer/enforcer.py

```diff
@@ -120,21 +119,23 @@ def enforce_single_project(enforcer, project_id, policy_filename):
 
 
 def main(argv):
+    """The main entry point for Forseti Security Enforcer runner."""
+
     del argv
 
     enforcer = initialize_batch_enforcer(
-            FLAGS.concurrent_threads, FLAGS.maximum_project_writer_threads,
-            FLAGS.maximum_firewall_write_operations, FLAGS.dry_run)
+        FLAGS.concurrent_threads, FLAGS.maximum_project_writer_threads,
+        FLAGS.maximum_firewall_write_operations, FLAGS.dry_run)
 
     if FLAGS.enforce_project and FLAGS.policy_file:
         enforcer_results = enforce_single_project(enforcer,
                                                   FLAGS.enforce_project,
                                                   FLAGS.policy_file)
 
-        print(enforcer_results)
+        print enforcer_results
```

Comment:
We should be adding:

from __future__ import absolute_import
from __future__ import division
from __future__ import print_function

to our imports and using print() to make the code easier to move to python3 support.